src/chromap.cc

```diff
@@ -1389,6 +1404,7 @@ void Chromap<MappingRecord>::MapPairedEndReads() {
   std::cerr << "Mapped all reads in " << Chromap<>::GetRealTime() - real_start_mapping_time << "s.\n";
   delete[] mm_history1;
   delete[] mm_history2;
+	//mm_to_candidates_cache.PrintStats();
```

Comment:
Can we remove this comment?

Reply:
The code mentioned above is all cleaned.